Dockerfile

```diff
@@ -39,4 +51,11 @@ RUN curl https://sdk.cloud.google.com | bash
 CMD ["/bin/bash"]
 
 # Copy the v2d project
-COPY ./ /finemapping
\ No newline at end of file
+COPY ./ /home/my-user/finemapping
+
+# Make all files in finemapping owned by the non-root user
+USER root
+RUN chown -R my-user:my-user /home/my-user/finemapping
+
+# Run container as non-root user
+USER my-use
```

Comment:
I suppose this should be `my-user`, right?

Reply:
yep